polyfills/Array/from/tests.js

```diff
@@ -87,32 +87,46 @@ describe('returns an array with', function () {
 			}
 		}
 
-		it('can convert from a user-defined iterator', function () {
-			function iterator(cnt) {
-				return {
-					next: function () {
-						return cnt === 0
-							? {
-								done: true
-							}
-							: {
-								value: cnt--,
-								done: false
-							};
+		it('can convert from a custom iterator', function () {
+			function iterable(count) {
+				var iterator = {
+					'@@iterator': function () {
+						throw new Error();
 					}
 				};
+				iterator[Symbol.iterator] = function () {
```

Comment:
I think you need to add `Symbol.iterator` as a dependency in the `config.json` for this polyfill otherwise this will throw in any browser which don't support `Symbol.iterator`.